src/index.js

```diff
@@ -333,17 +324,30 @@ function ncc (
             loader: eval('__dirname + "/loaders/uncacheable.js"')
           },
           {
-            loader: eval('__dirname + "/loaders/ts-loader.js"'),
+            loader: eval('__dirname + "/loaders/swc-loader.js"'),
             options: {
-              transpileOnly,
-              compiler: eval('__dirname + "/typescript.js"'),
-              compilerOptions: {
-                module: 'esnext',
-                target: 'esnext',
-                ...fullTsconfig.compilerOptions,
-                allowSyntheticDefaultImports: true,
-                noEmit: false,
-                outDir: '//'
+              minify: false, // TODO: maybe we could omit terser if `true`?
+              exclude: tsconfig.exclude,
+              sourceMaps: compilerOptions.sourceMap || false,
+              module: {
+                type: compilerOptions.module && compilerOptions.module.toLowerCase() === 'commonjs' ? 'commonjs' : 'es6',
+                strict: false,
+                strictMode: true,
+                lazy: false,
+                noInterop: !compilerOptions.esModuleInterop
+              },
+              jsc: {
+                externalHelpers: false,
+                keepClassNames: true,
+                target: compilerOptions.target && compilerOptions.target.toLowerCase() || 'es2021',
+                paths: compilerOptions.paths,
+                baseUrl: compilerOptions.baseUrl,
```

Comment:
@kdy1 The `paths` and `baseUrl` are failing with:

```
panicked at 'failed to get current directory: Error { kind: Unsupported, message: "operation not supported on this platform" }', /home/runner/work/swc/swc/ecmascript/transforms/module/src/path.rs:81:56

Stack:

Error: 
    at u.exports.__wbg_new_59cb74e423758ede (/Users/styfle/Code/vercel/ncc/dist/ncc/loaders/swc-loader.js.cache.js:1:2692)
    at null.<anonymous> (wasm://wasm/04480186:0:15193164)
    at null.<anonymous> (wasm://wasm/04480186:0:16752724)
    at null.<anonymous> (wasm://wasm/04480186:0:16566604)
    at null.<anonymous> (wasm://wasm/04480186:0:16677644)
    at null.<anonymous> (wasm://wasm/04480186:0:16749249)
    at null.<anonymous> (wasm://wasm/04480186:0:16736138)
    at null.<anonymous> (wasm://wasm/04480186:0:16687572)
    at null.<anonymous> (wasm://wasm/04480186:0:5322716)
    at null.<anonymous> (wasm://wasm/04480186:0:15233495)
```

Reply:
@kdy1 The new `@swc/wasm` no longer panics, but it still fails with very strange behavior.

Input:
```
import module from "@module";

console.log(module);
```

Output:
```
/******/ 	var __webpack_modules__ = ({

/***/ 105:
/***/ ((module) => {

module.exports = eval("require")("../../../../../../../../.././module");


/***/ })

/******/ 	});
```

Results in
```
Error: Cannot find module '../../../../../../../../.././module'
```